client/layout/index.jsx

```diff
@@ -167,10 +167,22 @@ class Layout extends Component {
 		colorSchemePreference: PropTypes.string,
 	};
 
+	constructor( props ) {
+		super( props );
+		this.state = {
+			isDesktop: isWithinBreakpoint( '>781px' ),
```

Comment:
We have to add the breakpoint to `mediaQueryOptions`. I'll push the changes.

Reply:
Added. https://github.com/Automattic/wp-calypso/pull/87702/commits/85517b76849e3f46d97eeb219f77bdbc09b2a325

Reply:
thanks!